great_expectations/expectations/expectation.py

```diff
@@ -2092,8 +2083,9 @@ def _validate(
             # Vacuously true
             success = True
         elif nonnull_count > 0:
-            success_ratio = float(nonnull_count - unexpected_count) / nonnull_count
-            success = success_ratio >= mostly
+            success = _mostly_success(
```

Comment:
This maintains the previous behavior that `success` can be `None` since it is assigned `None` by default. I can try to update this so `success` must be a `boolean` if that is desired. My preference would be to do that work in a separate PR since I haven't yet investigated ifthis involves changing other files.